samples/peripheral/radio_test/README.rst

```diff
@@ -56,6 +56,17 @@ The sample also requires one of the following testing devices:
    The radio test can be also performed using a spectrum analyzer.
    This method of testing is not covered by this documentation.
 
+nRF21540 Front-End-Module
+=========================
+
+The Radio Test sample supports nRF21540 Front-End-Module.
+
+This allows an increase of the transmitted power through a Power Amplifier (PA) or an increase of the sensitivity through a Low-Noise Amplifier (LNA).
+Any increase in power and sensitivity results in an increased communication range.
+The nRF21540 transmitted power gain, antenna and an activation delay can be configured using the user interface `User interface`_.
```

Comment:
Use active voice and fix the link:

You can configure the transmitted power gain, antenna, and activation delay in nRF21540 through the user interface, see :ref:`radio_test_ui`

Reply:
Ok. Done